test.js

```diff
@@ -242,7 +242,78 @@ test('keys and values should not be serialized', function (t) {
   })
 })
 
-test('iterators', function (t) {
+test('_close calls close for underlying store', function (t) {
+  t.plan(2)
+
+  var db = {
+    close: function (callback) {
+      t.pass('close for underlying store is called')
+      process.nextTick(callback)
+    }
+  }
+  var ld = new DeferredLevelDOWN(db)
+
+  ld.close(function (err) {
+    t.error(err, 'no error')
+  })
+})
+
+test('open error on underlying store calls back with error', function (t) {
+  t.plan(2)
+
+  var db = {
+    open: function (options, callback) {
+      t.pass('db.open called')
+      process.nextTick(callback, new Error('foo'))
+    }
+  }
+  var ld = new DeferredLevelDOWN(db)
+
+  ld.open(function (err) {
+    t.is(err.message, 'foo')
+  })
+})
+
+test('close error on underlying store calls back with error', function (t) {
+  t.plan(2)
+
+  var db = {
+    close: function (callback) {
+      t.pass('db.close called')
+      process.nextTick(callback, new Error('foo'))
+    }
+  }
+  var ld = new DeferredLevelDOWN(db)
+
+  ld.close(function (err) {
+    t.is(err.message, 'foo')
+  })
+})
+
+test('approximateSize is called on underlying store when store is open', function (t) {
```

Comment:
This is to test the "non-deferred" code path?

Reply:
Exactly

Reply:
Maybe I should rename the test to `non-deferred approximateSize`